vendor/vendor.json

```diff
@@ -347,10 +353,16 @@
 			"revisionTime": "2020-04-10T17:18:43Z"
 		},
 		{
-			"checksumSHA1": "Ola9GKVm7hRQO+Wworwy7zE16L0=",
+			"checksumSHA1": "oy1Jeoa2mt9xaoYsL44J6mt0+Mk=",
 			"path": "golang.org/x/sys/unix",
-			"revision": "12a6c2dcc1e4cb348b57847c73987099e261714b",
-			"revisionTime": "2020-02-12T08:54:49Z"
+			"revision": "6f6e22806c3450b5b4d275e186976d3b5758db41",
+			"revisionTime": "2021-09-06T14:56:19Z"
+		},
+		{
+			"checksumSHA1": "oy1Jeoa2mt9xaoYsL44J6mt0+Mk=",
+			"path": "golang.org/x/sys/internal/unsafeheader",
+			"revision": "6f6e22806c3450b5b4d275e186976d3b5758db41",
+			"revisionTime": "2021-09-06T14:56:19Z"
```

Comment:
I was wondering where this came from, but it was added as a dependency golang.org/x/sys/unix at revision d923437f in April 2020 where the current version vendored to secboot is from Feb 2020